src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -259,21 +261,25 @@ export default {
 
 <template>
 	<component :is="computedWrapperElement"
-		:id="wrapperId"
+		:id="wrapperId ?? (isButtonType ? id : null)"
 		:aria-label="isButtonType && ariaLabel ? ariaLabel : undefined"
-		:class="{
-			['checkbox-radio-switch-' + type]: type,
-			'checkbox-radio-switch--checked': isChecked,
-			'checkbox-radio-switch--disabled': disabled,
-			'checkbox-radio-switch--indeterminate': hasIndeterminate ? indeterminate : false,
-			'checkbox-radio-switch--button-variant': buttonVariant,
-			'checkbox-radio-switch--button-variant-v-grouped': buttonVariant && buttonVariantGrouped === 'vertical',
-			'checkbox-radio-switch--button-variant-h-grouped': buttonVariant && buttonVariantGrouped === 'horizontal',
-			'button-vue': isButtonType,
-		}"
 		class="checkbox-radio-switch"
-		:style="cssVars"
+		:class="[
+			$props.class,
+			{
+				['checkbox-radio-switch-' + type]: type,
+				'checkbox-radio-switch--checked': isChecked,
+				'checkbox-radio-switch--disabled': disabled,
+				'checkbox-radio-switch--indeterminate': hasIndeterminate ? indeterminate : false,
+				'checkbox-radio-switch--button-variant': buttonVariant,
+				'checkbox-radio-switch--button-variant-v-grouped': buttonVariant && buttonVariantGrouped === 'vertical',
+				'checkbox-radio-switch--button-variant-h-grouped': buttonVariant && buttonVariantGrouped === 'horizontal',
+				'button-vue': isButtonType,
+			},
+		].flat()"
+		:style="wrapperStyle"
```

Comment:
**nitpick:** to have less differences with `master`, keep original `cssVars` and use it here `:style="[style, cssVars]"`.

Or I didn't understand correctly why `cssVars` had to be replaced with `wrapperStyle`.

Reply:
No would work :)